modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -41,13 +45,17 @@ You can publish an API specification to Anypoint Exchange for use in API Manager
 
 == How API Designer Relates to Other MuleSoft Products
 
-API Designer is an integral part of the MuleSoft API-led approach to integrating applications. The following diagram depicts how the API specifications that you write in API Designer can be used in other MuleSoft products.
+API Designer is an integral part of the MuleSoft API-led approach to integrating applications. The following diagram shows how the API specifications that you write in API Designer can be used in other MuleSoft products.
```

Comment:
This reads like the text that should start off this topic set: it tells what API Designer is and how "you" use it.